README.md

```diff
@@ -43,10 +44,12 @@ def my_custom_command_handler(robot_session, message):
 
 
 robot_session_factory = RobotSessionFactory(
-    api_key="<YOUR_API_KEY>",
-    custom_command_callback=my_custom_command_handler
+    api_key="<YOUR_API_KEY>"
 )
 
```

Comment:
Why don't you add a small comment explaining what this is doing and what is the behavior of having both of them registered?

Reply:
done